lib/merkle_tree.ex

```diff
@@ -22,10 +22,13 @@ defmodule MerkleTree do
 
   defstruct [:blocks, :root, :hash_function]
 
-  @number_of_children 2 # Number of children per node. Configurable.
+  @number_of_children 2 # Number of children per node
```

Comment:
Since this functionality does not work at all anyway, maybe just drop it?

Reply:
We've already broken compatibility with original implementation by introducing fixed height.

Reply:
done